File: PLD1_InputProcessOutput/EX4_totalPurchase/totalPurchase.php
<?php
$tax = 0.06;

// Input
$item1 = readline("Enter a Price Item 1 : $");
$item2 = readline("Enter a Price Item 2 : $");
$item3 = readline("Enter a Price Item 3 : $");
$item4 = readline("Enter a Price Item 4 : $");
$item5 = readline("Enter a Price Item 5 : $");

// Process
$subTotal = $item1 + $item2 + $item3 + $item4 + $item5;
$salesTax = $subTotal * $tax;
$total = $subTotal + $salesTax;

// Output
print("subtotal  : " . $subTotal . "\n");
print("sales tax : " . $salesTax . "\n");
print("Total     : " . $total    . "\n");

File: PLD1_InputProcessOutput/EX4_totalPurchase/totalPurchase.py
item1 = int(input("Enter a Price Item 1 : $"))
item2 = int(input("Enter a Price Item 2 : $"))
item3 = int(input("Enter a Price Item 3 : $"))
item4 = int(input("Enter a Price Item 4 : $"))
item5 = int(input("Enter a Price Item 5 : $"))
tax = 0.06

subTotal = item1 + item2 + item3 + item4 + item5
salesTax = subTotal * tax
salesTax = int(salesTax)
total = subTotal + salesTax

print("Subtotal  : $", subTotal)
print("Sales tax : $", salesTax)
print("Total     : $", total)
